compiler/rustc_session/src/options.rs

```diff
@@ -1068,6 +1068,9 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "support compiling tests with panic=abort (default: no)"),
     parse_only: bool = (false, parse_bool, [UNTRACKED],
         "parse only; do not compile, assemble, or link (default: no)"),
+    partially_uninit_const_threshold: Option<usize> = (None, parse_opt_number, [TRACKED],
+        "allow generating const initializers with mixed init/uninit bytes, \
+        and set the maximum size at which this is allowed (default: never)"),
```

Comment:
Maximum size of what -- of the const allocation, or of one of the (un)init ranges? I tried understanding this from the code but failed...